packages/algorithm/rollup.config.js

```diff
@@ -1,40 +1,6 @@
-import nodeResolve from '@rollup/plugin-node-resolve';
-import sourcemaps from 'rollup-plugin-sourcemaps';
-import postcss from 'rollup-plugin-postcss';
+import { createRollupConfig } from '../../rollup.src.config';
```

Comment:
For consistency, perhaps the other two base Rollup config files should also output a config-generating function versus the config object itself?